backend/tests/courses/test_api.py

```diff
@@ -0,0 +1,208 @@
+from django.test import TestCase
+from django.urls import reverse
+from options.models import Option
+from rest_framework.test import APIClient
+
+from courses.models import Department, Instructor, Requirement
+from courses.util import create_mock_data, get_or_create_course
```

Comment:
If `create_mock_data` is only used to generate data for tests, can we move it into a util file in the tests directory?